### PR TITLE
remove ui modules

### PR DESCRIPTION
**Pull request by johnsonw**
remove ui modules

Signed-off-by: Will Johnson <william.c1.johnson@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
+      env: Type='Copr deploy'
+      before_deploy:
+        - openssl aes-256-cbc -K $encrypted_253525cedcf6_key -iv $encrypted_253525cedcf6_iv -in copr-key.enc -out copr-key -d
+        - curl -O https://raw.githubusercontent.com/m3t/travis_wait/master/travis_wait
+        - chmod 755 travis_wait
+      deploy:
+        skip_cleanup: true
+        provider: script
+        script:
+          - docker run -dit --name copr -v $(pwd):/builddir imlteam/dotnet-docker
```
Comment (jgrund) [inline .travis.yml]: This has to be a single command.

Comment (johnsonw) [reply .travis.yml]: I think I copied it before you changed it to a single command. I've updated now.

**Comment by jgrund** [comment]
Testing this locally I'm seeing:

```
Jun 20 17:44:48 admin.lfs.local systemd[1]: Starting IML Manager Target.
Jun 20 17:44:49 admin.lfs.local node[17755]: path.js:7
Jun 20 17:44:49 admin.lfs.local node[17755]: throw new TypeError('Path must be a string. Received ' + inspect(path));
Jun 20 17:44:49 admin.lfs.local node[17755]: ^
Jun 20 17:44:49 admin.lfs.local node[17755]: TypeError: Path must be a string. Received undefined
Jun 20 17:44:49 admin.lfs.local node[17755]: at assertPath (path.js:7:11)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Object.join (path.js:1217:7)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Object.<anonymous> (/usr/lib/node_modules/@iml/realtime/logger.js:16:14)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Module._compile (module.js:577:32)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Object.Module._extensions..js (module.js:586:10)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Module.load (module.js:494:32)
Jun 20 17:44:49 admin.lfs.local node[17755]: at tryModuleLoad (module.js:453:12)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Function.Module._load (module.js:445:3)
Jun 20 17:44:49 admin.lfs.local node[17755]: at Module.require (module.js:504:17)
Jun 20 17:44:49 admin.lfs.local node[17755]: at require (internal/module.js:20:19)
```

**Comment by jgrund** [comment]
Actually, I'm not seeing the realtime.service file anywhere?

**Comment by johnsonw** [comment]
Interesting, I wonder why it isn't producing the service file. Downloading on my end.

**Comment by jgrund** [comment]
> Interesting, I wonder why it isn't producing the service file. Downloading on my end. 

It looks like it was never added to this PR.

File: iml-realtime.service [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=IML Realtime Service
+PartOf=iml-manager.target
+After=iml-settings-populator.Service
```
Comment (jgrund) [inline iml-realtime.service]: The `.Service` should be lowercase.

File: iml-realtime.service [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Requires=iml-settings-populatore.Service
```
Comment (jgrund) [inline iml-realtime.service]: The .Service should be lowercase, and typo on populator.

Comment (johnsonw) [reply iml-realtime.service]: nice catch

File: iml-realtime.service [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+Type=simple
+EnvironmentFile=/var/lib/chroma/iml-settings.conf
+Environment=NODE_ENV=production
+ExecStart=/bin/node /usr/lib/node_modules/@iml/realtime/index.js
+Restart=on-failure
+StandardOutput=journal
+StandardError=journal
+
+[Install]
+WantedBy=iml-manager.target
```
Comment (jgrund) [inline iml-realtime.service]: I should have mentioned this, but this file already existed in the manager.

Use this as the base:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/7d319d369a82d7c4c539d12887866d0911dbb5f4/chroma-manager/iml-realtime.service#L1-L17

Comment (jgrund) [reply iml-realtime.service]: You'll obviously need to change the ExecStart to point at the new path, but the rest should be good.

Comment (johnsonw) [reply iml-realtime.service]: The only thing different here is `ExecStart`. But I think this is the path we want right? Do I need to change anything here?

**Pull request merged**